ttnn/cpp/ttnn/core.hpp

```diff
@@ -70,4 +70,22 @@ using core::get_memory_config;
 using core::has_storage_type_of;
 using core::pad_to_multiple_of_tile_size;
 using core::set_printoptions;
+
+
+extern std::atomic<std::int64_t> TENSOR_ID;
```

Comment:
is it possible to put all of it inside of the graph processor?
or you wthink we need it somewhere else?

Reply:
Tensor ID is used in the python code as well. And both C++ and python code need to use the same atomic variable.  Can we leave it as is on this PR and address it later?